assets/js/modules/analytics/datastore/__snapshots__/properties.test.js.snap

```diff
@@ -0,0 +1,668 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I'm not a huge fan of snapshots but this seemed the easiest way of checking the sort order!